src/components/GlobalKeyboardListeners.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import CommandBar from 'src/components/CommandBar';
+
+const GlobalKeyboardListeners: React.FC = () => {
+  return (
+    <>
+      <CommandBar />
```

Comment:
hmmm, a few thoughts here:

1. It seems to me that the commandbar component is not a keyboard listener. A keyboard listener would just dispatch an action to set the commandbar to be open but wouldn't have the actual component code.
2.  Was going to suggest rendering it conditionally but better do it inside of the component body, left the comment above
3. If we apply 1. should CommandBar be under app directly, similar to the feedback widget